site/content/docs/5.3/customize/options.md

```diff
@@ -20,6 +20,7 @@ You can find and customize these variables for key global options in Bootstrap's
 | `$enable-transitions`          | `true` (default) or `false`        | Enables predefined `transition`s on various components. |
 | `$enable-reduced-motion`       | `true` (default) or `false`        | Enables the [`prefers-reduced-motion` media query]({{< docsref "/getting-started/accessibility#reduced-motion" >}}), which suppresses certain animations/transitions based on the users' browser/operating system preferences. |
 | `$enable-grid-classes`         | `true` (default) or `false`        | Enables the generation of CSS classes for the grid system (e.g. `.row`, `.col-md-1`, etc.). |
+| `$enable-css-grid`             | `true` (default) or `false`        | Enables the experimental CSS Grid system (e.g. `.grid`, `.g-col-md-1`, etc.). |
```

Comment:
Small mistake here, it's `true` by default in our documentation, but `false` by default in the released library:

https://github.com/twbs/bootstrap/blob/6f87fce7962bef83c59eff895d891016db620cd7/scss/_variables.scss#L379

So I'm suggesting this change:

```suggestion
| `$enable-css-grid`             | `true` or `false` (default)        | Enables the experimental CSS Grid system (e.g. `.grid`, `.g-col-md-1`, etc.). |
```